core/common/src/main/java/alluxio/util/webui/UIMountPointInfo.java

```diff
@@ -43,6 +44,9 @@ public UIMountPointInfo(String mountPoint, MountPointInfo mountPointInfo) {
     mUfsUsedBytes = mountPointInfo.getUfsUsedBytes();
     mReadOnly = mountPointInfo.getReadOnly();
     mShared = mountPointInfo.getShared();
+    if (mountPointInfo.getProperties() != null) {
+      mProperties = mountPointInfo.getProperties().toString();
+    }
```

Comment:
if `mProperties` is final, it can be done in this way:

```suggestion
    mProperties = mountPointInfo.getProperties() != null ?
        mountPointInfo.getProperties().toString() : "";
    }
```

Reply:
but since other fields are not marked `final` (though they could), I am fine to leave it as is